web/src/pages/Upstream/List.tsx

```diff
@@ -136,12 +136,11 @@ const Page: React.FC = () => {
             <PlusOutlined />
             {formatMessage({ id: 'component.global.create' })}
           </Button>,
-          <Button type="primary" onClick={() => {
+          <Button type="default" onClick={() => {
```

Comment:
I think the original way looks better.
At the same time, If really need to do this, also need to modify the style of other pages.

Reply:
not sure if we really need the primary button here 🤔